client/common.go

```diff
@@ -199,10 +199,10 @@ func (x *contextCall) processResponse() bool {
 	// get result status
 	st := apistatus.FromStatusV2(x.resp.GetMetaHeader().GetStatus())
```

Comment:
What if we're to return `error` from `FromStatusV2`?

Reply:
Unfortunately not all statuses are errors. We have [this](https://github.com/nspcc-dev/neofs-sdk-go/blob/e9de866ce63ecce456da8249dde3f79afbe114c8/client/status/v2.go#L62) one. In fact it is success and doesn't implement error interface. I'm  worry we will have to add some conditions in code like - this error (I mean `SuccessDefaultV2` ) is not an real error and skip it

Reply:
That one is just `nil`.

Reply:
replaced with nil